admin/bin/qserv-data-loader.py

```diff
@@ -52,9 +52,9 @@
 
 """
 
-#--------------------------------
+# -------------------------------
```

Comment:
Extract of PEP-8 standard:

http://legacy.python.org/dev/peps/pep-0008/#comments
Each line of a block comment starts with a # and a single space

So it seems my editor is PEP-8 compliant on this point, isn't it?
And I would propose not to revert the file, indeed this is kind of detail.

Cheers

On 04/30/2015 05:31 PM, Andy wrote:
>
> In admin/bin/qserv-data-loader.py 
> <https://github.com/lsst/qserv/pull/67#discussion_r29439910>:
>
> > @@ -52,9 +52,9 @@
> >
> >  """
> >
> > -#--------------------------------
> > +# -------------------------------
>
> I believe your editor is misinterpreting PEP-8. Do not trust it. And 
> this is not a good reason to re-format the code because there may be 
> other editors which say something different.
>
> The only change to |qserv-data-loader.py| is copyright year change and 
> few random spaces. This change should not be needed. This file should 
> be reverted but I think you have already merged it.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/lsst/qserv/pull/67/files#r29439910>.
>